app/config.py

```diff
@@ -133,6 +134,9 @@ class Config(object):
     # PII check
     SCAN_FOR_PII = os.getenv("SCAN_FOR_PII", False)
 
+    # Documentation and Translation
```

Comment:
Translation?

Reply:
it was aspirational actually this comment, I wanted any future work, which I envision we would do in regards to offering english and french translations, to go here. I can remove and add it in later when we do

Reply:
as discussed on slack, we can follow up on this after. I was hoping translation constants can also be below this comment as well along with the DOCUMENTATION_DOMAIN. Currently only DOCUMENTATION_DOMAIN is defined but in the future others can be related to translation

Reply:
Can you remove the "Translation" part for now until we decide if we do further changes?